core/src/main/java/ru/urururu/sanity/rules/NullPointer.java

```diff
@@ -13,7 +13,7 @@
  */
 public class NullPointer {
     public void enforce(Cfg cfg, CallsMap callsMap) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "callsMap". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1172)